src/lib/redis.ts

```
@@ -1,16 +1,39 @@
-redis.on('connect', () => {
-  console.log('Redis connected');
-});
+const redisClientSingleton = () => {
+  return new Redis({
+    host: process.env.REDIS_HOST,
+    port: Number(process.env.REDIS_PORT),
+    password: process.env.REDIS_PASSWORD
+  });
+};
+
+declare const globalThis: {
```

Biome (1.9.4)

[error] Do not shadow the global "globalThis" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)